pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -38,14 +27,8 @@ buildPythonPackage rec {
   doCheck = false;
 
   nativeBuildInputs = [ pkgconfig which doxygen wxGTK ];
-
-  buildInputs = [ libjpeg libtiff SDL
-      gst-plugins-base libnotify freeglut xorg.libSM ncurses
-      requests libpng gstreamer libX11
-      pathlib2
-      (wxGTK.gtk)
-  ]
-    ++ lib.optional openglSupport pyopengl;
+  buildInputs = [ ncurses wxGTK.gtk ];
+  propagatedBuildInputs = [ numpy pillow six ];
```

Comment:
I don't see mention for why you've  added these three.

Reply:
See my last comment: https://github.com/NixOS/nixpkgs/pull/94108#issuecomment-667794537